Library/Formula/gnu-apl.rb

```diff
@@ -14,12 +14,11 @@ class GnuApl < Formula
   # GNU Readline is required; libedit won't work.
   depends_on "readline"
   depends_on :macos => :mavericks
+  depends_on "postgresql" => :recommended
```

Comment:
This may cause warnings. Some functions will be unavailable. So, I am afraid not.